src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -93,34 +93,36 @@
     <!-- /Why This Matters -->
 
     <!-- What You Can Do -->
-    <div class="vads-l-grid-container vads-u-padding-y--6 vads-u-padding-x--4 large-screen:vads-u-padding-x--0">
-      <div class="vads-l-row">
-        <div class="vads-l-col--12 medium-screen:vads-l-col--9">
-          <!-- Title -->
-          <p class="va-u-text-transform--uppercase vads-u-color--gray-medium vads-u-font-size--sm vads-u-margin-bottom--0">What you can do</p>
-          <h2 class="vads-u-margin--0">{{ fieldClpWhatYouCanDoHeader }}</h2>
-          <p class="va-introtext vads-u-margin-top--1 vads-u-margin-bottom--4">{{ fieldClpWhatYouCanDoIntro }}</p>
+    <div>
```

Comment:
I kinda wonder if it would help at all clarify we have this wrapper divs by adding a class like `.va-alernating-background-color-row` or something like that.

Reply:
That sounds smart! I'll add that now